doc/source/tutorial.rst

```diff
@@ -753,6 +753,13 @@ nodes may be different on your machine since the layout is not deterministic):
 
    Our social network with the Kamada-Kawai layout algorithm
 
+If you prefer to use `matplotlib`_ as a default plotting engine, create an axes and use
+the ``target`` argument:
+
+>>> import matplotlib.pyplot as plt
+>>> fig, ax = plt.subplots()
+>>> plot(g, layout=layout, target=ax)
```

Comment:
It seems to me that this does not make `matplotlib` the _default_. It is necessary to explicitly specify the `target` argument, i.e. modify the default.

That brings up the question: what if I want `matplotlib` to be the default? Should this be possible? Does the current design even allow making it the default, give that `target` takes an already constructed axes object? I just want a plot to show up in my notebook, and I want it to be made with matplotlib.

Keep in mind that these comments are coming from a Python novice who has very minimal experience with matplotlib. Take it as such: the perspective of a novice (which I think is useful, but should not always be used as the basis of decisions).

Reply:
(I'm also hijacking this a bit to try to learn a bit more about Python ;-) )

Reply:
Discussion continues below in the main comment section.